examples/sphere.py

```diff
@@ -83,13 +84,15 @@
 import numpy as np
 import tqdm
 
+# yapf: disable
 from ribs.archives import CVTArchive, GridArchive
 from ribs.emitters import (EvolutionStrategyEmitter, GaussianEmitter,
-                           GradientEmitter, GradientArborescenceEmitter,
-                           IsoLineEmitter)
+                           GradientArborescenceEmitter,
+                           GradientOperatorEmitter, IsoLineEmitter)
 from ribs.schedulers import BanditScheduler, Scheduler
 from ribs.visualize import cvt_archive_heatmap, grid_archive_heatmap
 
+# yapf: enable
```

Comment:
had to temporarily turn off yapf here to pass the commit check. yapf and isort have conflicting ways of doing imports from the multiple emitter classes here. can you please look into resolving the conflict?